src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/WebUIFactoryProvider.cs

```diff
@@ -27,8 +27,8 @@
 
 namespace Microsoft.IdentityModel.Clients.ActiveDirectory
 {
-    internal interface ICryptographyHelper
+    internal class WebUIFactoryProvider
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a 'protected' constructor or the 'static' keyword to the class declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://iddp-sonarqube:9000/coding_rules#rule_key=csharpsquid%3AS1118)